src/components/annual-report/section1/section1Calculations.js

```diff
@@ -717,13 +462,142 @@ const getSection1dChartInfo = ({ allOrgUnitsData, periodsIDs, ou }) => {
     return chartInfo
 }
 
+const getExpectedValues = ({ numerator, response, pe, ou }) => {
+    return numerator.dataSetID.reduce((totalExpected, dsUID) => {
+        totalExpected += Number(
+            getVal({ response, dx: dsUID + '.EXPECTED_REPORTS', ou, pe }) ?? 0
+        )
+        return totalExpected
+    }, 0)
+}
+
+const getActualValue1C = ({ response, dx, co, pe, ou }) => {
+    return getValCO({ response, dx, co, ou, pe })
+}
+
+const calculateSection1C = ({
+    expected_reports_over_all_org_units,
+    expected_reports_by_org_unit_level,
+    count_of_data_values_over_all_org_units,
+    count_of_data_values_by_org_unit_level,
+    mappedConfigurations,
+    period,
+    overallOrgUnit,
+    defaultCOC,
+}) => {
+    const section1C = []
+
+    // summarize expected reports
+    const overall_expected_reports = convertAnalyticsResponseToObject({
+        ...expected_reports_over_all_org_units,
+    })
+    const overall_counts = convertAnalyticsResponseToObject({
+        ...count_of_data_values_over_all_org_units,
+    })
+    const by_level_expected_reports = convertAnalyticsResponseToObject({
+        ...expected_reports_by_org_unit_level,
+    })
+    const by_level_counts = convertAnalyticsResponseToObject({
+        ...count_of_data_values_by_org_unit_level,
+    })
+
+    const metadata = [
+        expected_reports_over_all_org_units,
+        expected_reports_by_org_unit_level,
+        count_of_data_values_over_all_org_units,
+        count_of_data_values_by_org_unit_level,
+    ].reduce((allItems, resp) => {
+        return { ...allItems, ...resp.metaData.items }
+    }, {})
+
+    const subOrgUnits =
+        expected_reports_by_org_unit_level.metaData?.dimensions?.ou ?? []
+
+    for (const de in mappedConfigurations.dataElementsAndIndicators) {
+        //  get overall values
+        const numerator = mappedConfigurations.dataElementsAndIndicators[de]
+        const threshold = numerator.missing
+        const [deOperand = de, coOperand = defaultCOC] =
```

Comment:
just a naming suggestion from what we used in the data entry app: `deID` and `cocID`